examples/README.md

```diff
@@ -9,7 +9,7 @@ versions of InstrumentStudio.
 
 | InstrumentStudio Version | Release | Download |
 | ------------------------ | -------- | ------- |
-| 2024 Q3                  | [2.0.0](https://github.com/ni/measurement-plugin-python/releases/tag/2.0.0) | [measurement-plugin-python-examples-2.0.0.zip](https://github.com/ni/measurement-plugin-python/releases/download/2.0.0/measurement-plugin-python-examples-2.0.0.zip) |
+| 2024 Q3                  | [2.1.0](https://github.com/ni/measurement-plugin-python/releases/tag/2.1.0) | [measurement-plugin-python-examples-2.1.0.zip](https://github.com/ni/measurement-plugin-python/releases/download/2.1.0/measurement-plugin-python-examples-2.1.0.zip) |
```

Comment:
Similarly, should be a new 2024 Q4 section.

Reply:
Updated.

Reply:
Updated.